Neuron/Sections/Transaction/TransactionConfirmViewController.swift

```diff
@@ -182,7 +182,7 @@ class TransactionConfirmSendViewController: UIViewController {
     @IBAction func confirm(_ sender: Any) {
         let password = passwordTextField.text ?? ""
         if password.lengthOfBytes(using: .utf8) < 8 {
-            Toast.showToast(text: "请输入有效的钱包密码")
+            Toast.showToast(text: "密码不正确")
```

Comment:
Password length requirement should not be hard coded. Either use `PasswordValidator` or expose the length from that validator.

Reply:
Also think about this. If a user adds a wallet with current requirement (>= 8 characters), then the app changes that to 10 in the future, then the user won't be able to use that. So perhaps just check empty here?